### ISSUE TITLE
How to deal with overflow-y

### ISSUE THREAD

**Issue by jdang67** [valor-software/ng2-select#938]
The drop down works OK if the list is very short and the body enough space to show the drop-down.
Otherwise, most of data items is hidden from user. Can we change from div to select object.

**Comment by optimistex** [comment]
Hi. I continue developing the component in this fork: https://github.com/optimistex/ng2-select-ex
Please describe your expectations from the dropdown when the body hasn't enough free space by vertical.
And maybe I will can to implement it.

**Comment by jdang67** [comment]
Currently, ng2-select generates one Html tag for each <option> tag and those tags belong to the body of the page. As the result when the number of items is large,  the vertical scroll bar is added to allow us to see all items of the drop-down,  and in some cases, only few items will be visible at first.  You can reproduce this problem if the ng2-select is use in a small dialog.  
![ng2-select](https://user-images.githubusercontent.com/8005940/35629467-522f79b0-0664-11e8-9b07-d8f2dc444a09.png)

**Comment by optimistex** [comment]
Are you mean something like this?
![image](https://user-images.githubusercontent.com/11228640/35630328-33513742-06b2-11e8-8721-858a0f48ab4e.png)
__________________
Instead of this?
![image](https://user-images.githubusercontent.com/11228640/35630390-5800d41c-06b2-11e8-97ab-117f5e7a7964.png)

**Comment by jdang67** [comment]
Yes, making sure opening the drop down won't change the vertical spacing between the drop down and component below it.

**Comment by optimistex** [comment]
It's done. 
There is more details: https://github.com/optimistex/ng2-select-ex
